include/emp/bits/BitVector.hpp

```diff
@@ -701,6 +700,72 @@ namespace emp {
       return (int) (pos_found + (field_id * FIELD_BITS));
     }
 
+    /// Pop the last bit in the vector
+    bool PopBack() {
+      bool val = Get(num_bits-1);
+      if (val)  Set(num_bits-1, 0);
+      Resize(num_bits - 1);
+      return val;
+    }
+
+    /// Push given bit(s) onto the vector
+    void PushBack(bool bit=true, size_t num=1) {
```

Comment:
mark args in functions const  (ex const bool bit, const size_t num etc ) for all functions